### PR TITLE
Move attestation service instructions to main page

### PR DESCRIPTION
**Pull request by nambrot**
### Description

It was not obvious that you are supposed to run the attestation service, so this PR moves the instructions back onto the main page

### CODE REVIEW

File: packages/docs/getting-started/running-a-validator.md [hunk 1]
```diff
@@ -512,7 +515,108 @@ You can now run the node for the attestation service in the background:
 docker run --name celo-attestations --restart always -p 8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin --unlock $CELO_ATTESTATION_SIGNER_ADDRESS
 ```
 
-By now, you should have setup your Validator account appropriately. You can finish the actual deploy of the attestation service under the [Attestation Service at the documentation page](running-attestation-service.md).
+A part of that we are going to setup the following environment variable about the Attestation Service Docker image:
```
Comment (timmoreton) [inline packages/docs/getting-started/running-a-validator.md]: typos -- maybe `Next we will set up the Attestation Service itself. First, specify the following environment variables:`

also, could use some explanation that the "node" is different from the Attestation Service....

File: packages/docs/getting-started/running-a-validator.md [hunk 2]
```diff
@@ -512,7 +515,108 @@ You can now run the node for the attestation service in the background:
...
+
+```bash
+# On the Attestation machine
+export CELO_IMAGE_ATTESTATION="us.gcr.io/celo-testnet/celo-monorepo:attestation-service-8f87dda20f1854c9532a5cbcf8ff556f48dff413"
+# if you followed the instruction of setting up the attestation signer
+export CELO_PROVIDER=http://localhost:8545
+```
+
+### Sms Providers
```
Comment (timmoreton) [inline packages/docs/getting-started/running-a-validator.md]: `SMS`

File: packages/docs/getting-started/running-a-validator.md [hunk 3]
```diff
@@ -512,7 +515,108 @@ You can now run the node for the attestation service in the background:
...
+
+Currently the Sms providers supported are Nexmo & Twilio. You can create your user account in the provider of your election using the [Nexmo Sign Up form](https://dashboard.nexmo.com/sign-up) or the [Twilio Sign Up form](https://www.twilio.com/try-twilio).
```
Comment (timmoreton) [inline packages/docs/getting-started/running-a-validator.md]: `SMS`

Comment (timmoreton) [reply packages/docs/getting-started/running-a-validator.md]: `&` -> `and`

Comment (timmoreton) [reply packages/docs/getting-started/running-a-validator.md]: and let's give users a steer if there's a reason to do so

File: packages/docs/getting-started/running-a-validator.md [hunk 4]
```diff
@@ -512,7 +515,108 @@ You can now run the node for the attestation service in the background:
...
+
+| Variable                     | Explanation                                                     |
+| ---------------------------- | --------------------------------------------------------------- |
+| TWILIO_ACCOUNT_SID           | The Twilio account ID                                           |
+| TWILIO_MESSAGING_SERVICE_SID | The Twilio Message Service ID. Starts by `MG`                   |
+| TWILIO_AUTH_TOKEN            | The API authentication token                                    |
+| TWILIO_BLACKLIST             | A comma-sperated list of country codes you do not want to serve |
+
+After you signed up for Twilio at [https://www.twilio.com/try-twilio](https://www.twilio.com/try-twilio), you should see your `ACCOUNT SID` and your `AUTH_TOKEN` in the top right of the console. You'll also want to enter in a credit card to fund the account. For most text messages, the costs will be very low. Find a more comprehensive price list at [https://www.twilio.com/sms/pricing](https://www.twilio.com/sms/pricing). If there are countries that you do not want to serve, you can specify them with the `TWILIO_BLACKLIST`.
+
```
Comment (timmoreton) [inline packages/docs/getting-started/running-a-validator.md:521]: `with the TWILIO_BLACKLIST.` -> cut `the`

File: packages/docs/getting-started/running-a-validator.md [hunk 5]
```diff
@@ -512,7 +515,108 @@ You can now run the node for the attestation service in the background:
...
+```
+
+If everything goes well users should see that you are ready for attestations by running:
+
+```bash
+# On your local machine
+celocli account:get-metadata $CELO_VALIDATOR_ADDRESS
+```
+
+By now, you should have setup your Validator account appropriately.
```
Comment (timmoreton) [inline packages/docs/getting-started/running-a-validator.md:597]: Links to show metadata and attestation stats on blockscout?

**Review by timmoreton — COMMENTED** [review]
Also need to remove TOC entry from `SUMMARY.md`

**Comment by timmoreton** [comment]
nearly 👍

File: packages/docs/getting-started/running-a-validator.md (hunk 5 above)
Comment (nambrot) [reply packages/docs/getting-started/running-a-validator.md:597]: Beyond the cli account cmd we don't really have anything. And the stats will just show 0?

Comment (timmoreton) [reply packages/docs/getting-started/running-a-validator.md:597]: but you can let them know their metadata link will be visible there, and that the stats will update in the future

**Review by timmoreton — APPROVED** [review]

**Pull request merged**